v2/concordium/types.proto

```diff
@@ -1934,6 +1941,8 @@ enum TransactionType {
   TRANSFER_WITH_SCHEDULE_AND_MEMO = 18;
   CONFIGURE_BAKER = 19;
   CONFIGURE_DELEGATION = 20;
+  SUSPEND = 21;
```

Comment:
Will it be a transaction to suspend?

Reply:
My understanding is, that there should be a transaction to self-suspend. We could probably also do this via something like 'ConfigureBaker'.

Reply:
Right, good idea to do it as part of 'ConfigureBaker' as Thomas also mentions below :blush: